robot_assignments/web/jet2.robot

```diff
@@ -0,0 +1,97 @@
+*** Settings ***
+Library     SeleniumLibrary
+Library     String
+
+*** Variables ***
+${urlLogin}    http://125.26.15.143:13132/ 
+${urlhome}      http://125.26.15.143:13132/home.html
+${username1}    helloworld
+${password1}    1234
+${username2}    oz4899
+${password2}    1234
+${loginIDFieldXpath}    //input[@id='username']
+${loginPasswordFieldXpath}      //input[@id='password']
+${loginButtonXpath}     //button[@type='submit']
+${homePageContentXpath}     //div[@class='container']/div[@class='box']
+${searchFieldXpath}     //div[@class='column is-four-fifths']/input 
+${searchButtonXpath}       //button[@id='searchbtn'] 
+${itemListXpath}        //div[@class='card-content p-4']/p
+${noResultFoundXpath}       //div[@class='box']/div/div
+${firstItemXpath}       //div[@class='box']/div/div[1]
+${addToCartButtonXpath}      //button[@class='button is-info is-size-5']
+${itemQuantityXpath}     //span[@id='lblCartCount']/b
+${plusButtonXpath}      //div[@id='remove']/button
+${cartHoverXpath}       //div[@class='column']/div/div[@style='text-align: center;']/div/div[@class='dropdown-trigger']
+${clearCartButtonXpath}     //button[@class='button is-danger']
+
+*** Keywords ***
+Open Website
+    Open Browser  url=${urlLogin}  browser=chrome
+
+Login Website   
+    Input Text   ${loginIDFieldXpath}    ${username2}
+    Input Text   ${loginPasswordFieldXpath}      ${password2}
+    Click Element   ${loginButtonXpath}
+    Wait Until Page Contains Element    ${homePageContentXpath}
+    ${currentUrl}=     Get Location
+    Should be equal      ${urlhome}     ${currentUrl}
+
+*** Test cases ***
+Testcase 01 - Verify that user cannot login with wrong username
+    Open Website
+    Input Text   ${loginIDFieldXpath}     ${username1}
+    Input Text   ${loginPasswordFieldXpath}     ${password1}
+    Click Element   ${loginButtonXpath}
+    ${currentUrl}=     Get Location
+    Should be equal      ${urlLogin}     ${currentUrl}
+    sleep   1s
+    Close Browser
+
+Testcase 02 - Verify that user can login to the system when username are correct
+    Open Website   
+    Login Website
+    sleep   1s
```

Comment:
Sleep  1s -- Ok for 1st version.
Better solution is to apply wait until element on the page is visible so ensure that page is displayed as expected.
e.g. keyword
Verify home page is displayed
Wait Until Element Is Visible    ${homepage_element}    timeout=${timeout}

_locator = xpath of element display on home page only
timeout = can be 1s_